backend/main.py

```
@@ -361,6 +363,234 @@ def get_all_products(db: Session = Depends(get_db)):
+    if order.user_id != current_user.user_id:
+        raise HTTPException(status_code=403, detail="You do not have permission to view this order's bill")
+
+    bill = db.query(models.Bill).filter(models.Bill.order_id == order_id).first()
+    if not bill:
+        raise HTTPException(status_code=404, detail="Bill not found for this order")
+    return bill
+
+  
+  
+@app.get("/order/{order_id}/details", response_model=schemas.OrderDetailOut)
+def get_order_details(
```

Ruff (0.8.2)

Missing return type annotation for public function `get_order_details`

(ANN201)